src/components/editor-page/document-bar/permissions/permission-section-special-groups.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React, { Fragment, useMemo } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { useApplicationState } from '../../../../hooks/common/use-application-state'
+import { PermissionEntrySpecialGroup } from './permission-entry-special-group'
+import { AccessLevel, SpecialGroup } from './types'
+
+// Defined in https://github.com/hedgedoc/hedgedoc/blob/cbd2d3231/src/groups/groups.special.ts
+export const SPECIAL_GROUP_EVERYONE = '_EVERYONE'
+export const SPECIAL_GROUP_LOGGED_IN = '_LOGGED_IN'
```

Comment:
These are equivalent to the `SpecialGroup` enum. You could remove some redundant code in this component if you do the check in `PermissionEntrySpecialGroup` by using the enum.

Reply:
Removed duplicate constants.